README.md

```diff
@@ -79,12 +83,14 @@ coverage_percentage = 95
 allowed_mutants_percentage = 20
 # cyclomatic complexity allowed value
 cyclomatic_complexity_allowed = 10
+# minimum deadcode confidence
+min_deadcode_confidence = 100
 
 [ignore]
 # Comma seperated folders if more than one directory or leave empty after =
 cyclomatic_complexity_exclude =  C:\Projects\PythonRepo\python_sample\FunctionDefExtractor\test_resource, C:\Projects\PythonRepo\python_sample\FunctionDefExtractor\test
-# Comma seperated source folders if more than one directory or leave empty after =
-pylint_ignore = C:\Projects\PythonRepo\python_sample\FunctionDefExtractor\test
+# If there are ignores list them with absolute path with file name in a text file(line by line) or leave empty after =
```

Comment:
Thank you for introducing the pylint_ignore file! This will fit the use-case very well.